src/com/owncloud/android/lib/common/accounts/AccountUtils.java

```diff
@@ -218,7 +218,7 @@ public static void saveClient(OwnCloudClient client, Account savedAccount, Conte
 
 		if (client != null) {
 			String cookiesString = client.getCookiesString();
-			if (cookiesString != "") {
+			if (!cookiesString.equals("")) {
```

Comment:
Make this `!"".equals(cookiesString)` because if cookiesString happens to be null, you'll get `NullPointerException`